docs/src/pages/system/basics/basics.md

```diff
@@ -186,6 +186,14 @@ Cons:
 
   We believe that for most uses it's **fast enough**, but there are simple workarounds when performance becomes critical. For instance, when rendering a list with many items, you can use a CSS child selector to have a single "style injection" point (using d. for the wrapper and a. for each item).
 
+### API tradeoff
+
+In the previous versions, we supported the system properties as first class properties on the `Box` component. However, as we are extending the system to be a superset of CSS (contains all CSS properties/selectors in addition to custom ones) it's no longer an option to support all properties as first class props. This is the main reason why, we decided to support all system properties under one prop - `sx`, which in addition allows developers to define other CSS definitions.
```

Comment:
Matt might suggest to replace "we", not sure

```suggestion
In the previous versions, we supported the system properties as first-class properties on the `Box` component. However, as we are extending the system to be a superset of CSS (contains all CSS properties/selectors in addition to custom ones) it's no longer an option to support all properties as first-class props. This is the main reason why, we decided to support all system properties under one prop - `sx`, which in addition allows developers to define other CSS definitions.
```

Reply:
> Matt might suggest to replace "we", not sure 

I think I forget this one as well. @mbrookes explained it once but I kind of forgot :blush:

Reply:
I cannot get over this :D

Reply:
Can't be good at everything :wink: There's a reason technical writers exist.